### ISSUE TITLE
Question about how to extract loss for each epoch?

### ISSUE THREAD

**Issue by Joanne-PYL** [dvl-tum/mot_neural_solver#30]
Hi, 

Thank you for the amazing work. I am using the model training cells trajectories. I believe that the result I trained is overfitting. I am relatively new to deep learning algorithms. I am wondering that is there any way to extract training and validation loss for each epoch, so that I can adjust the parameters and layer numbers in the model? Thank you for your help.

Best,
Joanne

**Comment by Joanne-PYL** [comment]
Never mind, I found the solution haha.
